test/cljs/status_im/test/contacts/events.cljs

```diff
@@ -152,7 +154,7 @@
    load-contact-groups
    load-default-contacts (add-contact-groups, add-contacts, add-contacts-to-group ;TODO add-chat, load-commands!)
    add-contact-handler (add-new-contact-and-open-chat, status-im.contacts.events/add-new-contact,
-                        status-im.contacts.events/send-contact-request ;TODO start-chat)
+                        status-im.contacts.events/send-contact-request, status-im.chat.events.start-chat)
```

Comment:
status-im.chat.events **/** start-chat ?

Reply:
Probably I misunderstood the comment, 
I thought that it meant that the tests were implicitly testing this event: `status-im.chat.events.start-chat`